### DISCUSSION TITLE
failed to login,failed to init auth

### DISCUSSION THREAD

**Discussion by KeyToLove** [aws/codewhisperer-command-line-discussions#190] [Support ticket]
### Checks

- [X] I ran `q doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `q restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/q-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

lanuch desktop app and login.

### Actual behaviour

![image](https://github.com/aws/q-command-line-discussions/assets/54970106/97081234-79b5-4194-82b3-846c16f617b9)

i had try restart and re-install Q ,but none of them worked. 😢 

### Environment

```toml
q-details:
  - 1.0.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 10
  - mem: 16.00 GB
os-info:
  - macOS 14.5.0 (23F79)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/USER
  - exe-path: /Users/USER/.local/bin/q
  - install-method: unknown
  - env-vars:
    - PATH: /Users/USER/Library/pnpm:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/USER/Library/pnpm:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/USER/.local/bin
    - QTERM_SESSION_ID: 77cfb9224418444e900edf3f54134355
    - Q_SET_PARENT_CHECK: 1
    - Q_TERM: 1.0.0
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.googlecode.iterm2
```

**Comment by grant0417** [comment]
Hey @KeyToLove looks like you are using a slightly older version, can you update to the latest version here: https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-getting-started-installing.html

**Reply by KeyToLove** [reply]
> Hey @KeyToLove looks like you are using a slightly older version, can you update to the latest version here: https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-getting-started-installing.html

@grant0417  unfortunately，I just installed the latest version ( 1.1.0 ) via the link you provided, but the same problem still exists. :( 

![image](https://github.com/aws/q-command-line-discussions/assets/54970106/7c6313e7-5a9a-4174-b49b-58cfabb5d9d5)
